extensions/kill-process/src/index.tsx

```diff
@@ -88,23 +141,25 @@ export default function ProcessList() {
     >
       {state
         .filter((process) => {
-          if (query == null) {
+          if (query == "" || query == undefined) {
             return true;
           }
 
-          const nameMatches = process.name.toLowerCase().includes(query.toLowerCase());
+          const name = process.type == "aggregatedApp" ? process.appName : process.processName;
```

Comment:
When comparing, it's usually preferred to use strict compare.

Reply:
Got it, will fix it soon!